### DISCUSSION TITLE
QR code help

### DISCUSSION THREAD

**Discussion by NickPronto** [gopro/labs#92] [General]
I'm playing around with the latest firmware with my GoPro Hero 8, and have an issue. I'm using USB Power trigger, with Start Quick and end to shoot a clip and then repeat.

What I can't seem to figure out is how to keep the camera on between shots while still using the USB Power trigger to capture:

my code:
!uN!SQ!4E!30NQ!R

!uN - USB Power Trigger
!SQ - Start Quick
!4E - end in 4 seconds
!30NQ - do nothing for 30 seconds
!R - repeat action

after the 4 seconds of shooting, it turns the camera off and goes back to charging. 

When off, will the video upload to the cloud (provided it's still getting power from the USB)? It doesn't seem to be uploading with my subscription, but it could be just waiting to hit full charge.

**Comment by fxstein** [comment]
WiFi uploads only start when the camera is turned on and within reach of the network and not filming.

Maybe try a preset that has a time limit.

**Reply by NickPronto** [reply]
So a stacking preset to keep the camera on? 

I’d like to keep the camera on all the time and use just the usb power to trigger capture and stop capture, not to turn the camera on and off. 

Is there another "stop recording" action that doesn't turn the camera off? I'm looking here but only see "End":

https://gopro.github.io/labs/control/actions/

**Reply by dnewman-gpsw** [reply]
Currently there is no USB trigger that doesn't turn to camera on or off.  !uN was a very crude dashcam mode, it intend to power on the command, and !uE, to end capture and power off.  

The only way way to cloud upload from within a QR script is to use the !U command.

**Reply by NickPronto** [reply]
Awesome. I may need to just code it on the https api. Thank you!!!
